doc/src/manual/faq.md

```diff
@@ -556,6 +556,30 @@ julia> remotecall_fetch(anon_bar, 2)
 1
 ```
 
+## Troubleshooting "method not matched" (parametric invariance, etc.)
+
+### Why doesn't it work to declare `foo(bar::Vector{Real}) = 42` and then call `foo([1])`?
+
+As you'll see if you try this, the result is
+
+```jldoctest
+julia> foo(x::Vector{Real}) = 42
+foo (generic function with 1 method)
+
+julia> foo([1])
+ERROR: MethodError: no method matching foo(::Array{Int64,1})
+Closest candidates are:
+  foo(!Matched::Array{Real,1}) at REPL[2]:1
+```
+
+This is because `Vector{Real}` is not a supertype of `Vector{Int}`! You'd probably solve this problem with
+something like `foo(bar::Vector{T}) where {T<:Real}`. The `T` is a wild card; you first specify that it must
+be a subtype of real, then allow the function to take any vector of that type. If the static parameter `T`
```

Comment:
> any vector of that type

I read "any vector of type `T`", when the type is in fact `Vector{T}`... Maybe "any vector with elements of that type" or something similar?